pkg/minikube/bootstrapper/bsutil/kubeadm.go

```diff
@@ -63,6 +64,9 @@ func GenerateKubeadmYAML(cc config.ClusterConfig, n config.Node, r cruntime.Mana
 
 	cgroupDriver, err := r.CGroupDriver()
 	if err != nil {
+		if !r.Active() {
+			return nil, oci.ErrDockerRuntimeNotRunning
```

Comment:
runtime could be Docker or Containerd or Cri-o.
so it is not accurate to return "oci.ErrDockerRuntimeNotRunning"
and also the error should be in cruntime pacakge not oci (oci is only meant for docker and podman "drivers")

we can have a new error called
oci.ErrContainerRuntimeNotRunning